translations/combat.tex/pl.po

```diff
@@ -268,6 +363,24 @@ msgid ""
 "  \\hfill%\n"
 "  \\mbox{}%"
 msgstr ""
+"[h]\n"
+"  \\mbox{}%\n"
+"  \\hfill%\n"
+"  \\begin{minipage}[t]{0.415\\textwidth}\n"
+"    \\centering\n"
+"    \\includegraphics[width=\\linewidth]{\\examples/ranged_protected.png}\n"
+"  \\caption[halberdiers protected]{\\textit{Halabardnicy znajdujący się za wciąż stojącym Murem \\textbf{są chronieni} przed atakami z przeciwnej strony Muru.\n"
+"    Siła \\includesvg[height=10px]{\\svgs/unit_ranged.svg} ataku Złych Oczu jest \\textbf{zredukowana o 1}.}}\n"
```

Comment:
A tu nie chodzi o to, że obrażenia są obniżone o 1? Nie jestem pewien, czy na jedno wychodzi.

Reply:
lepiej nie dodawać nowych niejednoznaczności, zmieniłem